romancal/assign_wcs/pointing.py

```diff
@@ -45,3 +45,62 @@ def v23tosky(input_model, wrap_v2_at=180, wrap_lon_at=360):
     )
     model.name = "v23tosky"
     return model
+
+
+def dva_corr_model(va_scale, v2_ref, v3_ref):
+    """
+    Create transformation that accounts for differential velocity aberration
+    (scale).
+
+    Parameters
+    ----------
+    va_scale : float, None
+        Ratio of the apparent plate scale to the true plate scale. When
+        ``va_scale`` is `None`, it is assumed to be identical to ``1`` and
+        an ``astropy.modeling.models.Identity`` model will be returned.
+
+    v2_ref : float, None
+        Telescope ``v2`` coordinate of the reference point in ``arcsec``. When
+        ``v2_ref`` is `None`, it is assumed to be identical to ``0``.
+
+    v3_ref : float, None
+        Telescope ``v3`` coordinate of the reference point in ``arcsec``. When
+        ``v3_ref`` is `None`, it is assumed to be identical to ``0``.
+
+    Returns
+    -------
+    va_corr : astropy.modeling.CompoundModel, astropy.modeling.models.Identity
+        A 2D compound model that corrects DVA. If ``va_scale`` is `None` or 1
+        then `astropy.modeling.models.Identity` will be returned.
+
+    """
+    if va_scale is None or va_scale == 1:
+        return Identity(2)
+
+    if va_scale <= 0:
+        raise ValueError("'Velocity aberration scale must be a positive number.")
```

Comment:
Should we set it to 1 in this case for the moment?  Otherwise we need to rebuild all of the regression files, I think?  We can change this to an error in the future.